### ISSUE TITLE
why line coverage is displayed as a badge?

### ISSUE THREAD

**Issue by aviralpostman** [MishaKav/jest-coverage-comment#90]
Is there any particular reason to show only line coverage as a badge?
Also, in this [PR](https://github.com/MishaKav/jest-coverage-comment/pull/40) for relative coverage, why are we displaying the change in line coverage only and why not for others?

**Comment by MishaKav** [comment]
There are now particular reason, just because devs love badges :)
If more people will vote for this, I think It could be an option to enrich the config.

**Comment by aviralpostman** [comment]
ah, ok. I got confused there. Googled if line coverage is superior to others 😭.

Anyways, I've a feature request too: Can we have a flag to compare the coverage change wrt to the main branch? 
This would require to clone the main branch => run tests there => use the PR linked in the description to show the coverage difference.
Lmk if this feature is already in works.

**Comment by MishaKav** [comment]
Relative coverage has been requested several times, currently, I don't have enough time to develop this by myself, but there is initiated PR #40, so someone will, based on this, suggest a PR for this feature.

**Comment by MishaKav** [comment]
@aviralpostman Can you clarify your request? You want to show the coverage in `text` instead of `badge`?

**Comment by aviralpostman** [comment]
Yes, I wanted to know why line coverage was displayed as a badge and others as text. From you [replies](https://github.com/MishaKav/jest-coverage-comment/issues/90#issuecomment-2152913464), it felt like a personal preference.

I've changed the same in my fork, so all is good. Closing the issue.
<img width="829" height="524" alt="Image" src="https://github.com/user-attachments/assets/00e620a9-17ac-4bd4-b76f-33fa02888951" />
